src/bun.js.zig

```diff
@@ -535,6 +550,27 @@ pub noinline fn failWithBuildError(vm: *jsc.VirtualMachine) noreturn {
     Global.exit(1);
 }
 
+fn isBrokenEmulator() bool {
+    if (bun.Environment.isX64) {
+        const newBits = asm volatile (
+            \\
+            // copy input into xmm0
+            \\movsd %xmm2, %xmm0
+            // convert doubles in xmm0 to ints in xmm1
+            // -- this SHOULD leave xmm0 unchanged, but it does not in some Rosetta versions
+            \\cvttpd2dq %xmm0, %xmm1
```

Comment:
```suggestion
            \\movsd %%xmm2, %%xmm0
            // convert doubles in xmm0 to ints in xmm1
            // -- this SHOULD leave xmm0 unchanged, but it does not in some Rosetta versions
            \\cvttpd2dq %%xmm0, %%xmm1
```

For consistency with the standard library (lib/std/os/linux/x86_64.zig consistently uses `%%`) and compatibility with other backends (the C backend generates invalid code unless you use `%%`).